examples/smooth.py

```diff
@@ -82,9 +82,7 @@ def plot_raw_and_smooth_timeseries_and_psd(
         )
 
         # interpolate data to remove NaNs in the PSD calculation
-        pos_interp = interpolate_over_time(
-            pos, print_report=False, fill_value="extrapolate"
-        )
+        pos_interp = interpolate_over_time(pos, fill_value="extrapolate")
```

Comment:
Leaving `print_report` to its default `False` value is good here, but elsewhere in the `smooth.py` example I'd like you to explicitly add `print_report=True` to all the other filter/interpolation function calls, to replicate the previous behaviour, which is desired.
 
 The only possible exception is the final section of the smoothing example, where we apply multiple filters one after the other.  Feel free to leave `print_report` to its default `False` value there.